Material.Blazor/Foundation/IBatchingJsRuntime.cs

```diff
@@ -1,10 +1,29 @@
-﻿using System.Threading.Tasks;
+﻿using Microsoft.JSInterop;
+using System.Threading.Tasks;
 
 namespace Material.Blazor.Internal
 {
-    public interface IBatchingJsRuntime
+    /// <summary>
+    /// A service that wraps IJSRuntime, batching calls to InvokeVoidAsync().
+    /// </summary>
+    public interface IBatchingJSRuntime
     {
-        Task<T> InvokeAsync<T>(string identifier, params object[] args);
+        /// <summary>
+        /// Same as <see cref="JSRuntimeExtensions.InvokeVoidAsync(IJSRuntime, string, object[])"/>, except calls are batched in 20ms intervals.
```

Comment:
Classic case of docs are out of sync with code. The interval is actually 10ms at the moment.